content/code-generation-from-the-ast.md

```diff
@@ -0,0 +1,287 @@
++++
+author = ["Ian Eyberg"]
+date = "2015-03-13T12:50:47-07:00"
+linktitle = "Code Generation From the AST"
+title = "Code Generation From the AST"
++++
+
+[Deprehend](https://github.com/deferpanic/deprehend "Deprehend") is a tool
+that auto-generates go src to do things like wrapping goroutines with
```

Comment:
Suggestion: ~~to do things like wrapping~~ to wrap

The precedence isn't entirely clear to me. Is it? 

1. (goroutines with panic handlers) and (http handlers for metrics tracking)
2. (goroutines with panic handlers and http handlers) for metrics tracking